docs/pages/setup/admin/github-sso.mdx

```diff
@@ -33,10 +33,17 @@ Instructions for creating a GitHub OAuth app are available here:
 
 [Creating an OAuth App](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app)
 
-## Step 2/3. Create a GitHub authentication connector
+## Step 2/5. Define the GitHub authentication connector
```

Comment:
```suggestion
## Step 2/5. Define a GitHub authentication connector
```
The first paragraph of the section uses "a", and we haven't specified a GitHub connector yet.